src/app/modules/form/public-form/public-form.controller.ts

```diff
@@ -274,18 +276,20 @@ export const handleGetPublicForm: ControllerHandler<
           return res.json({ form: publicForm, isIntranetUser })
         })
     case FormAuthType.SGID_MyInfo: {
-      const accessTokenCookie = req.cookies[SGID_MYINFO_COOKIE_NAME]
-      if (!accessTokenCookie) {
+      const { jwt: accessToken = '', sub = '' } = JSON.parse(
+        req.cookies[SGID_MYINFO_COOKIE_NAME] ?? '{}',
+      )
```

Comment:
Could this potentially be dangerous? If someone passes a request with a malformed cookie that is not a JSON value, `JSON.parse` would fail and terminate the process. Maybe we should extract the `JSON.parse` step and wrap it in a try-catch block.